src/RootedTrees.jl

```diff
@@ -1413,6 +1415,42 @@ function butcher_representation(t::RootedTree, normalize::Bool = true)
     return result
 end
 
+"""
+    elementary_differential(t::RootedTree)
+
+Returns the elementary differential as a LaTeXString.
+
+"""
+function elementary_differential(t::RootedTree)
+    return latexstring(rec_elementary_differential(t))
+end
+
+"""
+Function used to go recursively through the RootedTree to generate the elementary differential of the tree.
+"""
+function rec_elementary_differential(t::RootedTree)
```

Comment:
```suggestion
# Function used to go recursively through the rooted tree 
# to generate the elementary differential of the tree.
function rec_elementary_differential(t::RootedTree)
```

It is somewhat common agreement in the Julia community that something becomes part of the public API if it is documented (and not explicitly documented as internal functionality). Thus, I suggest to change this docstring into a comment since `rec_elementary_differential` should not be not part of the public API.